go/mysql/replication.go

```diff
@@ -93,14 +93,14 @@ func (c *Conn) WriteComBinlogDumpGTID(serverID uint32, binlogFilename string, bi
 		4 + // data-size
 		len(sidBlock) // data
 	data, pos := c.startEphemeralPacketWithHeader(length)
-	pos = writeByte(data, pos, ComBinlogDumpGTID)             // nolint
-	pos = writeUint16(data, pos, flags)                       // nolint
-	pos = writeUint32(data, pos, serverID)                    // nolint
-	pos = writeUint32(data, pos, uint32(len(binlogFilename))) // nolint
-	pos = writeEOFString(data, pos, binlogFilename)           // nolint
-	pos = writeUint64(data, pos, binlogPos)                   // nolint
-	pos = writeUint32(data, pos, uint32(len(sidBlock)))       // nolint
-	pos += copy(data[pos:], sidBlock)                         // nolint
+	pos = writeByte(data, pos, ComBinlogDumpGTID)
+	pos = writeUint16(data, pos, flags)
+	pos = writeUint32(data, pos, serverID)
+	pos = writeUint32(data, pos, uint32(len(binlogFilename)))
+	pos = writeEOFString(data, pos, binlogFilename)
+	pos = writeUint64(data, pos, binlogPos)
+	pos = writeUint32(data, pos, uint32(len(sidBlock)))
+	pos += copy(data[pos:], sidBlock) //nolint
```

Comment:
```suggestion

```

@mattlord this suppressed linter err is an `ineffassign`, because `pos` is never used after this increment on line 103, so it's useless

We can just drop this line but I wanted a 2nd set of 👀, perhaps to see if you know what was intended here. Press the button if it looks good!